src/tests/Microsoft.Agents.BotBuilder.Dialogs.Tests/SkillDialogTests.cs

```diff
@@ -403,6 +429,99 @@ void CaptureAction(string toBotId, string toBotResource, Uri toUri, Uri serviceU
             Assert.Equal(1, (dialogOptions.ConversationIdFactory as SimpleConversationIdFactory).CreateCount);
         }
 
+        [Fact]
+        public async Task ContinueDialogAsync_ShouldReturnEndOfTurnOnValidateActivity()
+        {
+            _context.SetupGet(e => e.Activity)
+                .Returns(new Activity { Text = "shouldNotValidate" })
+                .Verifiable(Times.Exactly(1));
```

Comment:
Just confirm that this test is named correctly. The `shouldNotValidate` text makes me wonder...

Reply:
Yea, i had trouble picking up the name, because for normal behavior the validation should return true, but for this case it should be false. I'll change it to a better name